src/main/java/com/deloitte/elrr/datasync/KafkaAdminConfiguration.java

```diff
@@ -14,12 +14,14 @@ public class KafkaAdminConfiguration {
     /**
      *
      */
-    private static final int KAFKA_PARTITIONS = 6;
+    //private static final int KAFKA_PARTITIONS = 6; // PHL
+    private static final int KAFKA_PARTITIONS = 1;   // PHL
```

Comment:
I would make these property-configurable from application properties files, because it will vary.

Then in your local dev setup instructions you can just document that the applicable property should be set to 1 for a local install.